packages/wasm/src/index.ts

```diff
@@ -0,0 +1,232 @@
+//@ts-ignore source map not found
+import {
+  BlsVariant,
+  EcdsaVariant,
+  InitOutput,
+  //@ts-ignore source map not found
+  getModule,
+  initSync,
+} from './pkg/wasm-internal';
+export type { BlsVariant, EcdsaVariant } from './pkg/wasm-internal';
+
+import * as wasmInternal from './pkg/wasm-internal';
+
+let loadingPromise: Promise<any> | null = null;
+let wasmSdkInstance: InitOutput | undefined;
+
+// Give us a promise that _just_ encapsulates initializing the modules so we can wrap it in other logic
+async function initWasm() {
+  await initSync(getModule());
```

Comment:
```suggestion
  return initSync(getModule());
```

I think we need to return the SDK instance here so that we set `wasmSdkInstance` to the instance on L44 below?